File: client/src/app/shared/models/donation.ts

export interface IDonation {
    id: number;
    donationTypeName: string;
    donationShortName: string;
    remarks: string; 
    status: string; 
    setOn: Date;
}
 
export class Donation implements IDonation {
    id: number=0;
    donationTypeName: string;
    donationShortName: string;
    remarks: string; 
    status: string=null; 
    setOn: Date;
}

File: Core/Entities/Donation.cs
namespace Core.Entities
{
    public class Donation:BaseEntity
    {
        public string DonationTypeName { get; set; }
        public string DonationShortName { get; set; }
        public string Remarks { get; set; }
        public string Status { get; set; }
        
    }
}